modules/dataquery/readme/readme.md

```diff
@@ -0,0 +1,78 @@
+
+## Other Notes
+The most closely related module to the DQT is the Data Release module, which is used to show snapshots of data for any given release. 
+
+## Permissions
+Users must have `dataquery_view` permission to use the DQT. It’s important to note that more granular permissions do not exist for this module, to restrict access to a subset of the data. Anyone with the `dataquery_view` permission has access to the DQT, and therefore has access to all data in the LORIS instance. 
+
+## Interactions with LORIS
+The CouchDB import scripts (those named CouchDB_Import_*.php in the LORIS tools directory) must be run to import data from MySQL to CouchDB before the DQT can be used. It is recommended to set up a cronjob to execute these scripts to refresh the data on a regular basis. 
+
+## Configuration
+Use of the DQT module requires setting up the CouchDB server and credentials:
+
+[CouchDB](http://couchdb.apache.org)
+[Erica](https://github.com/benoitc/erica)
+
+Note: Your version of Erlang should be the most recent version that is compatible with both your CouchDB and Erica.
+
+Once these are installed, there are two ways to complete setup. See below:
+
+#### The Easy Way (for non-developers)
+
+1. create a database on your local CouchDB instance
+2. clone the code from the server [http://couchdb.loris.ca:5984/dataquerytool-$VERSION]
+..* ensure $VERSION is separated by underscores rather than dots (because dots are not allowed in CouchDB database names)
+..* i.e.
+`curl -H 'Content-Type: application/json' -X POST http://$YOURCOUCHDBADMIN:$YOURCOUCHADMINPASS@$YOURSERVERNAME:5984/_replicate -d '{"source":"http://couchdb.loris.ca:5984/dataquerytool-1_0_0", "target":"$YOURDATABASENAME"}'`
+
+#### The Hard Way (for developers)
+
+1. create a CouchDB database using Futon [http://127.0.0.1:5984/_utils/index.html]
+..* In the following example, the database is named "dqg".
+2. clone this repository `git clone git@github.com:aces/Data-Query-Tool.git`
+3. push to CouchDB using erica `cd Data-Query-Tool`
+`erica push http://adminuser:adminpass@127.0.0.1:5984/dqg`
+4. visit [http://127.0.0.1:5984/dqg/_design/DQG-2.0/_rewrite/] to ensure code was pushed
+
+Finally, there are further steps required to complete configuration, regardless of how the user completed installation and setup:
+
+Amend the section of your LORIS config.xml
+`<CouchDB>
+    <SyncAccounts>true</SyncAccounts>
+    <database>dqg</database>
+    <hostname>localhost</hostname>
+    <port>5984</port>
+    <admin>adminuser</admin>
+    <adminpass>adminpass</adminpass>
+</CouchDB>`
+
+To load the Data Query Tool with data stored in LORIS, run the `CouchDB_Import_*` scripts, in tools/ directory: 
+`cd $lorisroot/tools
+
+#Import the base candidate data
```

Comment:
I think the rendering for this formatting is not working properly. I think you need a space after the `#` and maybe a new line between the title and the text.